UnitTests/View/TextTests.cs

```diff
@@ -811,41 +784,16 @@ public void View_Draw_Horizontal_Simple_TextAlignments (bool autoSize)
         ((FakeDriver)Application.Driver).SetBufferSize (width + 2, 6);
 
         // frame.Width is width + border wide (20 + 2) and 6 high
-
-        if (autoSize)
```

Comment:
In the `public void View_Draw_Vertical_Simple_TextAlignments (bool autoSize)` unit test you maintained the `autoSize` parameter. Why did you remove it here in the horizontal test?

Reply:
Oops. Thanks.